### ISSUE TITLE
Add data URI favicon support

### ISSUE THREAD

**Issue by EatonZ** [11ty/api-indieweb-avatar#10]
If your site uses an SVG as its favicon, the leaderboard doesn't display it. You should add support for it.
Example entry in the `html` `head`:
`<link rel="icon" type="image/svg+xml" href="data:image/svg+xml,blah">`

**Comment by zachleat** [comment]
Yeah! This is also listed as #5 on the TODO list https://github.com/11ty/api-indieweb-avatar#indieweb-avatar-api

Test case from https://joshcrain.io/

**Comment by zachleat** [comment]
For some reason I feel like this is somehow related https://fav.farm/%F0%9F%8C%B6 though I’m not 100% sure

**Comment by zachleat** [comment]
~~also `<link rel="icon" type="image/x-icon" href="https://devographics.github.io/surveys/state_of_js/2022/images/js2022-favicon.svg">` from stateofjs.com~~
